docs/connectors/postgresql.asciidoc

```diff
@@ -1019,7 +1050,7 @@ The following configuration properties are _required_ unless a default value is
 
 |`time.precision.mode`
 |`adaptive`
-a| Time, date, and timestamps can be represented with different kinds of precisions, including: `adaptive` (the default) captures the time and timestamp values exactly as in the database uses either millisecond, microsecond, or nanosecond precision values based on the database column's type; or `connect` always represents time and timestamp values using Kafka Connect's built-in representations for Time, Date, and Timestamp, which uses millisecond precision regardless of the database columns' precision. See link:#temporal-values[temporal values].
+| Time, date, and timestamps can be represented with different kinds of precisions, including: `adaptive` (the default) captures the time and timestamp values exactly as in the database uses either millisecond, microsecond, or nanosecond precision values based on the database column's type; or `adaptive_time_microseconds` captures the date, datetime and timestamp values exactly as in the database uses either millisecond, microsecond, or nanosecond precision values based on the database column's type, except TIME type fields are always captured as microseconds;; or `connect` always represents time and timestamp values using Kafka Connect's built-in representations for Time, Date, and Timestamp, which uses millisecond precision regardless of the database columns' precision. See link:#temporal-values[temporal values].
```

Comment:
See above the same section above, e.g. s/uses/using/.

Reply:
done

also changed all other occurrences according to s/When time.precision.mode/When the time.precision.mode/